index.js

```diff
@@ -304,22 +318,24 @@ HtmlWebpackPlugin.prototype.postProcessHtml = function (html, assets, assetTags)
  */
 HtmlWebpackPlugin.prototype.addFileToAssets = function (filename, compilation) {
   filename = path.resolve(compilation.compiler.context, filename);
-  return Promise.props({
-    size: fs.statAsync(filename),
-    source: fs.readFileAsync(filename)
-  })
+  return Promise.all([
```

Comment:
I'm fine this that, but **if** we want to keep the same, we could use: https://github.com/sindresorhus/p-props

Reply:
That seems nice and small, and doesn't have any deps itself.